.github/workflows/ci-build.yml

```diff
@@ -32,6 +32,14 @@ jobs:
           distribution: 'adopt'
           java-version: '17'
 
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: '3.1'
+
+      - name: Install Licensee
+        run: gem install licensee
```

Comment:
Add new dependency to README / Contributing docs

Reply:
DId it